### PR TITLE
added: Playlist Cards and Pages UI

### PR DESCRIPTION
**Pull request by rkapoor10**
The Following Basic Structure is added along with UI

- Playlist Card with hover effects
- Playlist Page
- Explore Section with filter chips
- Horizontal Video Card
- Watch Later Page
- Liked Page
- History Page

Go through the deploy preview to see all this in action

### CODE REVIEW

File: src/components/playlists/Playlist.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const PlaylistContainer = () => {
+  const demoPlaylist = [
+    "Playlist 1",
+    "Playlist 2",
+    "Playlist 3",
+    "Playlist 4",
+    "Playlist 5",
+    "Playlist 6",
+    "Playlist 7",
+  ];
```
Comment (JantuDeb) [inline src/components/playlists/Playlist.jsx:13]: Each playlist should contain the videos in that playlist

File: src/components/shared/sidemenu/SideMenu.jsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ const SideMenu = () => {
             to={path}
             key={name}
             className="menu-item"
```
Comment (JantuDeb) [inline src/components/shared/sidemenu/SideMenu.jsx:37]: You can do this without the style prop, style prop will add `inline style`
```
.menu-item.active{
    color: "red",
    borderBottom:"0px"
    borderBottom: "0px",
}

```
The above code will work fine without `inline` style

File: src/router/Router.jsx [hunk 3]
```diff
@@ -16,8 +18,9 @@ const Router = ()=>{
             <Route path="/home" element={<HomePage/>}/>
             <Route path="/history" element={<HistoryPage/>}/>
             <Route path="/explore" element={<VideoListingPage/>}/>
-            {/* what will be better path for watch later ? */}
+            <Route path="/playlist" element={<PlaylistPage/>}/>
             <Route path="/watchlater" element={<WatchLaterPage/>}/>
+            <Route path="/liked" element={<LikedVideosPage/>}/>
             <Route path="/auth" element={<AuthPage/>} />
             <Route path="*" element={<ErrorPage/>} />
```
Comment (JantuDeb) [inline src/router/Router.jsx:27]: This is good

**Review by JantuDeb — APPROVED** [review]
This is looks good

File: src/components/playlists/Playlist.jsx (hunk 1 above)
Comment (rkapoor10) [reply src/components/playlists/Playlist.jsx:13]: I'll edit it as array of objects while adding functionality

File: src/components/shared/sidemenu/SideMenu.jsx (hunk 2 above)
Comment (rkapoor10) [reply src/components/shared/sidemenu/SideMenu.jsx:37]: ok

File: src/router/Router.jsx (hunk 3 above)
Comment (rkapoor10) [reply src/router/Router.jsx:27]: thanks

File: src/components/filters/filters.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.5rem;
+}
+
+.chip{
+    font-size: 1rem;
+    background-color: var(--darkgray-color);
+    /* this will be on active chip or chips if multiple selections */
+    /* color: var(--basered-color); */
```
Comment (sushil-mod) [inline src/components/filters/filters.css:16]: if it is not in use remove it

File: src/router/Router.jsx [hunk 5]
```diff
@@ -7,6 +7,8 @@ import HistoryPage from "../pages/HistoryPage"
 import VideoListingPage from "../pages/VideoListingPage"
 import WatchLaterPage from "../pages/WatchLaterPage"
 import AuthPage from "../pages/AuthPage"
+import LikedVideosPage from "../pages/LikedVideos"
+import PlaylistPage from "../pages/PlaylistPage"
```
Comment (sushil-mod) [inline src/router/Router.jsx:11]: this is also correct. just a suggestion you can try barrel export :-  https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno

**Review by sushil-mod — APPROVED** [review]
UI looking nice . just a suggestion . Overall code is good

File: src/router/Router.jsx (hunk 5 above)
Comment (rkapoor10) [reply src/router/Router.jsx:11]: sure thanks

File: src/components/filters/filters.css (hunk 4 above)
Comment (rkapoor10) [reply src/components/filters/filters.css:16]: yes

**Pull request merged**